clients/template-variable-client/README.md

```diff
@@ -0,0 +1,43 @@
+# `@epilot/template-variables-client`
```

Comment:
Directory should match the name of the client package

`clients/template-variables-client`